src/MongoDB.EntityFrameworkCore/Storage/MongoDatabaseWrapper.cs

```diff
@@ -187,7 +191,41 @@ private IEnumerable<MongoUpdate> ConvertUpdateEntriesToMongoUpdates(IList<IUpdat
 
     private static MongoUpdate ConvertAddedEntryToMongoUpdate(string collectionName, string id, IUpdateEntry entry)
     {
-        throw new NotImplementedException(); // EF-17
+        var entityType = entry.EntityType;
+        var entitySerializer = (IBsonDocumentSerializer)EntitySerializer.Create(entityType);
```

Comment:
This is the serializer that will determine what the documents in the database look like.

This is the same serializer we pass to the LINQ provider (it *has* to be the same one, or LINQ queries won't get translated correctly).